.gitignore

```diff
@@ -18,3 +18,6 @@ packages/**/dist
 packages/**/esm
 
 published.txt
+
+# Editor settings
+.vscode
```

Comment:
was this intended?

Reply:
Yes it was. There are no vscode files in the codebase and I needed a couple of settings set up which might not be the same for everyone